lib/puppet/provider/service/rcng.rb

```diff
@@ -0,0 +1,50 @@
+Puppet::Type.type(:service).provide :rcng, :parent => :bsd do
+  desc <<-EOT
+    RCng service management with rc.d
+  EOT
+
+  defaultfor :operatingsystem => [:netbsd, :cargos]
```

Comment:
note to self, need to review and make sure the confine of the parent applies to the `rcng` provider.